subsys/bluetooth/controller/hal/cntr.h

```diff
@@ -4,6 +4,7 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#include <stdint.h>
```

Comment:
```suggestion
```

Include `#include <stdint.h>` in c files where needed. This is not right to have here to satisfy files that include `#include "hal/cntr.h"`

That said, I will have a future action item on myself to clean up other redundant/incorrect includes in controller hal folder, like sys/dlist.h etc  which is never used in cntr.c file.

Reply:
Missed this comment.  There is a uint8_t and a uint32_t usage six lines down in this same file (the arguments to cntr_cmp_set()).  There is not way for this to compile correctly in all cases without this include here.